config/src/fmt.rs

```diff
@@ -18,6 +18,8 @@ pub struct FormatterConfig {
     pub func_attrs_with_params_multiline: bool,
     /// Style of quotation marks
     pub quote_style: QuoteStyle,
+    /// Style of underscores in number literals
+    pub number_literal_underscore: NumberLiteralUnderscore,
```

Comment:
small nit/side comment: maybe `number_literal_format` or `number_format` is better?

Reply:
I renamed it to `number_underscore` so that the config options `number_underscore = "remove"` and `number_underscore = "thousands"` are obvious and read well